libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts

```diff
@@ -49,6 +46,10 @@ describe('FormFieldComponent', () => {
     it('should have correct margin', () => {
       expect(spectator.element).toHaveComputedStyle({ 'margin-bottom': size('s') });
     });
+    it('should have elevation', () => {
```

Comment:
Consider readability:
```suggestion

    it('should have elevation', () => {
```

Reply:
Moved (and added spacing)